en/advanced_features/traffic_avoidance_adsb.md

```diff
@@ -94,7 +94,61 @@ The code can be found in `Navigator::check_traffic` ([/src/modules/navigator/nav
 PX4 will also forward the transponder data to a GCS if this has been configured for the MAVLink instance (this is recommended).
 The last 10 Digits of the GUID is displayed as Drone identification.
 
+## Testing/Simulated ADSB Traffic
+
+You can simulate ADS-B traffic for testing.
+Note that this requires that you [Build PX4](../dev_setup/building_px4.md)
+
+:::note
+Simulated ADS-B traffic can trigger real failsafe actions.
+Use with care in real flight!
+:::
+
+To enable this feature:
+
+1. Uncomment the code in `AdsbConflict::run_fake_traffic()`([AdsbConflict.cpp](https://github.com/PX4/PX4-Autopilot/blob/main/src/lib/adsb/AdsbConflict.cpp#L342C1-L342C1)).
+1. Rebuild and run PX4.
+1. Execute the [`navigator fake_traffic` command](../modules/modules_controller.md#navigator) in the [QGroundControl MAVLink Shell](https://docs.qgroundcontrol.com/master/en/analyze_view/mavlink_console.html) (or some other [PX4 Console or MAVLink shell](../debug/consoles.md), such as the PX4 simulator terminal).
+
+The code in `run_fake_traffic()` is then executed.
+You should see ADS-B warnings in the Console/MAVLink shell, and QGC should also show an ADS-B traffic popup.
+
+By default `run_fake_traffic()` publishes a number of traffic messages (it calls [`AdsbConflict::fake_traffic()`](#fake-traffic-method) to emit each report).
+These simulate ADS-B traffic where there may be a conflict, where there won't be a conflict, as well as spamming the traffic buffer.
+
+::: details Information about the test methods
```

Comment:
FYI this is a "details" twistie. It basically means that this information that isn't relevant to most users is hidden unless you click it.

![image](https://github.com/PX4/PX4-user_guide/assets/5368500/b7b7d109-3aa8-40d0-81e9-666e8f5ca0ad)

Reply:
hmm, this was the way then. Gotcha!